### ISSUE TITLE
Incorrect logical def for: pair of interacting organisms of the same species

### ISSUE THREAD

**Issue by cmungall** [PopulationAndCommunityOntology/pco#87]
```
'pair of interacting organisms'
 and (has-member only (member-of exactly 1 'species as a collection of organisms'))
```

The logical def is not correct

consider the pair human1 and mouse1. Both are members of exactly 1 species. So the pair satisfies the definition and classifies under this class.

Warning: metaclasses are hard to do right in OWL, trying to recapitulate metaclasses using mereology only gets you so far

In this particular case you can define a property chain and then say

`= has-member-member-of exactly 1 species`

**Comment by cmungall** [comment]
Same problem with single-species collection of organisms

(RO_0002351 only 'organism or virus or viroid')
 and (RO_0002351 only (RO_0002350 exactly 1 'species as a collection of organisms'))
 and (RO_0002351 min 2 'organism or virus or viroid')

**Comment by cmungall** [comment]
Note also for species as a collection of organisms you can state

organism SubClassOf member-of exactly 1 species

also add

'collection of organisms' SubClassOf has-member only organism (which I assume is the intent - you're recapitulating basic sets here, yes?)

**Comment by ramonawalls** [comment]
Thanks, @cmungall ! I'm on it.

**Comment by ramonawalls** [comment]
> Note also for species as a collection of organisms you can state
> 
> organism SubClassOf member-of exactly 1 species
> 
> also add
> 
> 'collection of organisms' SubClassOf has-member only organism (which I assume is the intent - you're recapitulating basic sets here, yes?)

I added the second axiom, but the first one is for a CARO term. I fear that adding axioms to it will cause problems with any ontologies that end up importing it via PCO.

**Comment by cmungall** [comment]
Still not fixed?

The current logical definition is:

'collection of organisms'
 and ('has member' exactly 2 'organism or virus or viroid')

which doesn't match the text def:

A collection of organisms that consists of exactly two organism, viruses, or viroids that are interacting with each other.

I think it may be impossible to provide an OWL definition for this class, so the equivalence axiom should be removed

what is the use case for this class? maybe it is safest to exclude it?

**Comment by ramonawalls** [comment]
Yeah, I have tried to find an owl definition that works, but without success. The equivalency axioms do a nice job with building the hierarchy, but as you say, they don't match the text definition.

I remember there was a use case for this when it was added, but I don't remember what it is now. Guess it is a good idea to document use cases when new terms are created. Regardless, I hesitate to remove it, because I do think it is being use. Perhaps I will just stick with the text definition and subclass axioms.
